packages/sdk-react-native/src/contexts/TurnkeyContext.tsx

```diff
@@ -83,414 +100,350 @@ export const TurnkeyProvider: FC<{
   const [session, setSession] = useState<Session | undefined>(undefined);
   const [client, setClient] = useState<TurnkeyClient | undefined>(undefined);
 
-  const expiryTimeoutRef = useRef<NodeJS.Timeout | null>(null);
+  // map to track expiration timers for each session by sessionId
```

Comment:
seriously nonblocking/tiny nit: we can use `session id` here instead of `sessionId` for consistency (I think `session id` is used more commonly in the codebase/this diff). fine with either, just optimizing for consistency